app/src/app/initApp.js

```diff
@@ -72,19 +73,25 @@ export default () => {
       }
     )
 
+    // workaround to solve invisible app problem when windows was closed on
+    // another monitor previously but Grape starts without the monitor now
+    if (state.dimensions.x < 0 || state.dimensions.x < 0) {
+      state.prefs = Object.assign(
+        state.prefs,
+        {x: 0, y: 0}
+      )
```

Comment:
You don't need  `state.prefs = `. I am assuming you wanted to set state.prefs.x and state.prefs.y. Also I would put it into some namespace, prefs.x is not clear what those coordinates are.

Reply:
`new BrowserWindow(state.prefs)` needs `x` and `y` in the object supplied at frist argument. how would you do it?